packages/ion_identity_client/lib/src/auth/services/create_recovery_credentials_service.dart

```diff
@@ -1,68 +1,86 @@
-// SPDX-License-Identifier: ice License 1.0
```

Comment:
Need to add a License header to all files

Reply:
Added license to all files without a license (skipped `.g.dart` files)